tsdb/errors/errors.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The etcd Authors
+// Copyright 2020 The Prometheus Authors
```

Comment:
Why are you changing the author?

Reply:
Because we rewrote completely the code? (:

Reply:
happy to revert